src/app/shared/helpers/date-time.ts

```diff
@@ -84,6 +92,39 @@ export class DateTime {
     return date.moment.diff(this.moment, Duration.SECOND) > 0;
   }
 
+  isDuring(range: DateRange) {
```

Comment:
how about:

isDuring(range: DateRange) {
  const today = new Date();
  const dateRange = (() => {
    switch (range) {
      case DateRange.TODAY:
        return moment(today).subtract(1, 'day');
      case DateRange.WEEK:
        return moment(today).subtract(1, 'week');
      case DateRange.FORTNIGHT:
        return moment(today).subtract(2, 'weeks');
      case DateRange.NINETY_DAYS:
        return moment(today).subtract(90, 'days');
      case DateRange.ONE_YEAR:
        return moment(today).subtract(1, 'year');
      case DateRange.EIGHTEEN_MONTHS:
        return moment(today).subtract(18, 'months');
      default:
        return null;
    }
  })();

  return this.moment.isSameOrAfter(dateRange);
}